CONTRIBUTING.md

```diff
@@ -0,0 +1,10 @@
+
+### Contributing code
+
+If you have improvements to Drake, send us your pull requests! For those
+just getting started, Github has a [howto](https://help.github.com/articles/using-pull-requests/).  Please make sure you look at our [Developer's page](http://drake002.csail.mit.edu/drake/sphinx/developers.html) for information on code style, etc, and be prepared to engage in active code review on your pull request.
+
+We would also like to hear about your success stories if you've used Drake in your own projects.  Please consider contributing to our [gallery](http://drake002.csail.mit.edu/drake/sphinx/gallery.html) by editing gallery.html in the drake/doc/ directory and submitting a pull request.
+
+Important note: Drake is an open source project licensed under extremely flexible terms intended to encourage use by anyone, for any purpose. When you make a contribution to the Drake project, you are agreeing to do so under those same terms. The details are below; if you aren't comfortable with those terms, we're still friends but you shouldn't contribute.
```

Comment:
You’re right. I did that too fast. And the details are split/duplicated in the
sphinx documentation. Will open a PR which cleans it up (CONTRIBUTING.md just
points to the sphinx doc).